io.github.ddanilov.soundscape.yaml

```diff
@@ -0,0 +1,26 @@
+app-id: io.github.ddanilov.soundscape
+runtime: org.kde.Platform
+runtime-version: '6.5'
+sdk: org.kde.Sdk
+command: soundscape
+finish-args:
+  # directory access
+  - --filesystem=host
+  # X11
+  - --device=dri
+  - --share=ipc
+  - --socket=x11
+  # audio
+  - --socket=pulseaudio
+rename-appdata-file: soundscape.metainfo.xml
+rename-desktop-file: soundscape.desktop
+rename-icon: soundscape
+modules:
+  - name: soundscape
+    buildsystem: cmake
```

Comment:
cmake-ninja?

Reply:
can also be `-ninja`, I do not see any difference there

Reply:
then use it, it's faster. `cmake` only exist as a fallback